src/front/styles/footer.css

```diff
@@ -0,0 +1,35 @@
+.page-footer {
+  background-color: #ffb74d !important;
+  color: white;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+}
+
+a {
+  color: white;
+}
+
+#signature {
+  font-size: 13px;
+}
+
+.fa {
+  width: 50px;
+  height: 50px;
+}
+
+#links {
+  padding-bottom: 45px;
```

Comment:
I think you did this to add height to the footer, no?

If it was the case, i'd suggest you to add a `height` attribute over the `page-footer`, and change this `padding-bottom` to a `margin-bottom` instead 🤓